vimdoc/regex.py

```diff
@@ -211,14 +211,23 @@ def _DelimitedRegex(pattern):
   # MATCH GROUP 1: The Name
   (
     # Non-commas or escaped commas or escaped escapes.
-    (?:[^\\,]|\\.)+
+    # Must not end with a space.
+    (?:[^\\,<]|\\.)+\S
   )
   # Optional identifier
   (?:
     # Separated by comma and whitespace.
-    ,\s*
+    \s*,\s*
```

Comment:
Seems unnecessary to me accepting extra whitespace before the comma here. Is that intentional?